src/lib/RequestParser.js

```diff
@@ -188,9 +185,15 @@ class RequestParser { // eslint-disable-line no-unused-vars
      * @returns {Nimiq.Address}
      */
     parseAddress(address, name) {
+        if (address === 'CONTRACT_CREATION') {
+            return Nimiq.Address.CONTRACT_CREATION;
+        }
```

Comment:
For a HTLC creation transaction to work, the recipient must be passed as the _exact_ object reference to `Nimiq.Address.CONTRACT_CREATION`. This can obviously not be done through RPC, so a constant is used and replaced here with the correct object reference.